test/Db/AbstractDbTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zf2 for the canonical source repository
+ * @copyright Copyright (c) 2005-2015 Zend Technologies USA Inc. (http://www.zend.com)
+ * @license   http://framework.zend.com/license/new-bsd New BSD License
+ */
+
+namespace ZendTest\Validator\Db;
+
+use ZendTest\Validator\Db\TestAsset\ConcreteDbValidator;
+use Zend\Db\Sql\Select;
+
+/**
+ * @group      Zend_Validator
+ */
+class AbstractDbTest extends \PHPUnit_Framework_TestCase
+{
+
+    protected $validator;
+
+    public function setUp()
+    {
+        $this->validator = new ConcreteDbValidator([
```

Comment:
A better approach would be to use mocking for the DB adapter. I'll see if I can work up an example.